### DISCUSSION TITLE
20240704會議

### DISCUSSION THREAD

**Discussion by ChiuAlfredo** [ChiuAlfredo/options_day_data#4] [會議討論]
7/4 開會內容
0.宣布短期目標
1.學習如何使用github
2.提出問題及驗證流程確定
3.討論7/3觀察到的東西
4.提出要給小佩或維中的問題
5.檢討簡報裡被砲過的東西
6.重做（gamma的計算和相關係數那邊）
7.補完新策略的資料（獲利那些）

**Comment by Will150143** [comment]
# 會議記錄

## 0. 短期目標
- 以學習學長策略作為短期目標
- 假如有餘韻再試著發展新策略

## 1. 學習GitHub功能
- 學習如何使用GitHub Discussions & Projects功能
- 後續補教學影片（程式先不急）
- 學習資源：
  - [what is github](https://learn.microsoft.com/en-us/training/modules/introduction-to-github/)
  - [如何使用project](https://learn.microsoft.com/en-us/training/modules/manage-work-github-projects/)
  - [markdown語法](https://learn.microsoft.com/en-us/training/modules/communicate-using-markdown/)

## 2. 提出問題及驗證流程
- A. 東西要邊做邊丟，可以一起看
  - a. 申請OneDrive
  - b. 如何分工接力（待定）

## 3. 討論7/3觀察到的東西
- A. 選擇權結算價計算
- B. 盤勢觀察會更新在 #3 
- C. 試著看成交單價格
- D. 學習看盤軟體操作
- E. discussion區分可能有用或純發現

## 4. 提出要給小佩或維中的問題
- A. 寄信時要寄給所有人
- B. 給小佩的問題在下面留言
- C. 維中的部分等後面實作時才比較會有問題

## 5. 檢討簡報裡被砲過的東西
- A. Share手寫資料（如：公式 用平板無邊際白板)
- B. 需要佐證（呈現統計結果)
- C. 使用資料說明要更詳細（如時間跟標的)
- D. 簡報設計要更直覺（使用名詞、要標單位、格式跟排版)
- E. 圖表資料要完整（縱軸標題)
- F. 在報告前一個禮拜開始做簡報（假如報告前一週有五個發現，就只報告五個，第六個可以繼續做但不報告)
*簡報先做好再繼續做第六個
- G. 正式報告前至少要試講兩次

## 6. 重做
- Gamma的計算
- 相關係數那邊

## 7. 補完新策略的資料（該議程刪除)
- 獲利相關資料

## 8. 臨時動議
- @Will150143 在提出議程時要標可以先放的內容（如問題跟被砲過的東西)
- 拿掉新策略（不報告)
- 下一步（各自開Todo)
  -檢驗波動幅度 @ChiuAlfredo  
  -價格波動方向驗證（期貨、指數) @Will150143 
  -價格波動&時間的關係 @Will150143 
  -選擇權的價性 @ChiuAlfredo 
  -探討成交量 @TsuYiii
- 學長回答內容下次開會處理 
- 下次開會時間待定

**Comment by ChiuAlfredo** [comment]
要做的事：
* 申請OneDrive
* 看github教學影片
* 要問的問題（小佩)：
  1.7/3號當天開盤開高，後續，可能會使用哪些策略
  2.如何獲取高頻資料、使用哪些資料以及如何驗證自己的猜想
  3.我們有觀察到最後一小時的價外P有漲有跌，請問學長當時做了哪些操作，是否方便說明您的策略
  4.Gamma值要怎麼在看盤軟體看（我們是用三竹)
* todo list
